weeks/week03/README.md

```diff
@@ -406,4 +406,4 @@ Write a series of functions to accomplish the following, building a larger progr
 6. After doing all of the above steps, create a new record with `ID`, `Name`, `Area Code`, `Height In Inches` and separate them with commas
 7. Combine all these processed records into a new CSV formatted string, with rows separated by `\n`.
 
-A sample solution is provided in [solution.js](solution.js).
\ No newline at end of file
+A sample solution is provided in [solution.js](solution.js).
```

Comment:
I'm unclear what change has been made here, it seems to be identical.  Perhaps I'm misreading?